packages/bbui/src/Form/Core/DatePicker.svelte

```diff
@@ -35,6 +39,11 @@
     if (newValue) {
       newValue = newValue.toISOString()
     }
+    // if time only set date component to today
+    if (timeOnly) {
+      const todayDate = new Date().toISOString().split("T")[0]
+      newValue = `${todayDate}T${newValue.split("T")[1]}`
```

Comment:
I don't think returning a time only works particularly well, because we would need to make an allowance everywhere else in the system for the "time" type not being a valid date. I tried this originally passing it around just as say "00:00:00" but a lot of places weren't happy with it being an invalid date, like flatpickr etc. Decided it was safer to pass it around as a full date, the database knows to strip it to the time component if a `TIME` type is selected, and we can just strip it as needed in the frontend.

Reply:
Sounds good 👍